components/Navbar.vue

```diff
@@ -399,6 +389,8 @@ export default class NavbarMenu extends mixins(
 
   beforeDestroy() {
     window.removeEventListener('scroll', this.onScroll)
+    this.setBodyScroll(true)
+    document.documentElement.classList.remove('is-clipped-touch')
```

Comment:
The last question. Where does the class name `is-clipped-touch` come from?

Reply:
B-dropdown
This class is being put on the html element when one of the drop downs is opened
Like the create dropdown for example

And it stays there after click and navigation

Reply:
Got it.